### DISCUSSION TITLE
I can not change arrow  color and dots  color

### DISCUSSION THREAD

**Discussion by nemofalcon** [ivylaboratory/angular-carousel#71] [Q&A]
For example  i use  this css code. but It has not changed
```
.carousel-arrow {
    background-color: lightblue;
    background-position: 50%;
    background-repeat: no-repeat;
    background-size: 31px;
    border-radius: 100px;
    box-shadow: 0 0 5px rgb(0 0 0 / 15%);
    cursor: pointer;
    height: 40px;
    margin-top: -20px;
    position: absolute;
    top: 50%;
    width: 40px;
    z-index: 10;
}
```

**Comment by cocoswebdevelop** [comment]
Were you able to fix this problem yet? Im running into it as well

**Reply by cocoswebdevelop** [reply]
@nemofalcon  if you want to override the css styling from the module create a <style> tag in your index.html. Not an elegant solution but it works!

`    <style>
        .carousel-dot{
          border: 2px solid black!important;
          border-radius: 100px;
          display: inline-block;
          height: 8px;
          margin: 4px;
          width: 8px;
        }
        .carousel-dot-active{
          background-color: black!important;
          border-radius: 100px;
          display: inline-block;
          height: 8px;
          margin: 4px;
          width: 8px;
        }
    </style>`

**Comment by NealRM** [comment]
Has anyone resolved this for the arrows? I can change the CSS in the devtools, but not from the CSS sheet for the Angular modular.
